model/project_vars.go

```diff
@@ -216,15 +221,36 @@ func (projectVars *ProjectVars) GetRestrictedVars() map[string]string {
 	return restrictedVars
 }
 
-func (projectVars *ProjectVars) GetAdminOnlyVars() map[string]string {
+// GetAdminOnlyVars will change to GetVars on EVG-16045 after removing restricted vars.
+func (projectVars *ProjectVars) GetAdminOnlyVars(t *task.Task) (map[string]string, error) {
 	adminOnlyVars := map[string]string{}
-
-	for k, v := range projectVars.Vars {
-		if projectVars.AdminOnlyVars[k] {
-			adminOnlyVars[k] = v
+	if evergreen.IsSystemActivator(t.ActivatedBy) {
+		for k, v := range projectVars.Vars {
+			if projectVars.AdminOnlyVars[k] {
+				adminOnlyVars[k] = v
+			}
+		}
+	} else if t.ActivatedBy != "" {
+		u, err := user.FindOneById(t.ActivatedBy)
+		grip.Error(err)
```

Comment:
Let's log with message.Fields and include some additional information (taskId, activatedBy, etc)

Reply:
done